docs/source/getting_started/installing.rst

```diff
@@ -92,6 +99,22 @@ The ``vncauthproxy`` service is installed and can be started::
 
     $ sudo service vncauthproxy start
 
+systemd
+~~~~~~~
+
+For systems running ``systemd``, a basic systemd script has been provided by Adam at
+the Georgia Tech Information Security Center. It has been tested on Debian 8.
+
+Copy the file ``scripts/vncauthproxy/init-systemd`` to
+``/lib/systemd/system/vncauthproxy.service`` and enable the service::
+
+    $ sudo cp /path/to/gwm/scripts/vncauthproxy/init-systemd /lib/systemd/system/vncauthproxy.service
+    $ sudo systemctl enable vncauthproxy
+
+The script does not currently support variables, so if you want VNC AuthProxy
```

Comment:
We talked about this in person, seems binary location must be absolute, other options are variable.